### DISCUSSION TITLE
LinkByRequestId

### DISCUSSION THREAD

**Discussion by paneq** [RailsEventStore/rails_event_store#1176] [Ideas]
Similar to `LinkByCausationId` etc

Should use the same metadata name that we automatically enrich in Rails env.

RailsEventStore/rails_event_store#382 RailsEventStore/rails_event_store#72 RailsEventStore/rails_event_store#221 RailsEventStore/rails_event_store#346

**Comment by mostlyobvious** [comment]
http://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-request_id
